backend/Dockerfile

```
@@ -0,0 +1,82 @@
+# 3. Generate Dart Frog build output (creates build/bin/server.dart)
+RUN dart_frog build
+
+# 4. AOT compile the generated server to a native binary
+RUN dart compile exe build/bin/server.dart -o build/bin/server
+
+# ── Stage 2: Runtime ────────────────────────────────────────
+# Use debian:bookworm-slim (NOT scratch) — the Flutter build image does not
+# provide /runtime/ like dart:stable does. AOT binaries need libc + friends.
+FROM debian:bookworm-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)